lib/general.js

```diff
@@ -44,12 +45,21 @@ class NodeVersionCheck extends DoctorCheck {
 }
 checks.push(new NodeVersionCheck());
 
+
+async function getNpmCmdPath () {
+  if (isWindows) {
```

Comment:
Call the function?

Also, this could be a good place for the ternary operator.